app/app/vhosts/index/controller.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+  provisionedDomains: Ember.computed.filter('model', function(vhost) {
+    return vhost.get('isProvisioned');
+  }),
```

Comment:
This (and `pendingDomains` and `deprovisionedDomains`) can be even terser: `Ember.computed.filterBy('model', 'isProvisioned')`